source/appModules/outlook.py

```diff
@@ -435,6 +435,15 @@ def _get_name(self):
 				flagIcon=0
 			flagIconLabel=oleFlagIconLabels.get(flagIcon)
 			if flagIconLabel: textList.append(flagIconLabel)
+			# Replied or forwarded state for this message is available from the object's value.
+			# We must parse this value correctly, as it contains redundant information.
+			# The several states are localized and separated by a space.
+			# Example output: 'Message Replied Read'
+			valueParts = self._get_value().split(" ")
+			# The first valuePart is the type of the selection, e.g. Message, Contact.
+			# The last valuePart indicates whether the message is read or unread.
+			if len(valueParts)>=3:
+				textList.extend(valueParts[1:-1])

```

Comment:
Can we be sure that the first part (message, contact etc) is never more than one word in all languages?

Reply:
Good question, which also applies to read/unread I belief.

Though JAWS uses these translated strings, I"m pretty sure they do some string matching on it, as the read state is ignored when Both Outlook and JAWS are set to Dutch, but the state is preserved when Outlook is Dutch and JAWS is English. So that's definitely not a clean solution.

I belief both Message and Unread are part of the NVDA translation, so it should be doable to check whether these words contain spaces in other languages without intervention of every single translator.